README.md

```diff
@@ -1,6 +1,6 @@
 [Sangria](http://sangria-graphql.org/) [MessagePack](http://msgpack.org/) marshalling.
 
-[![Build Status](https://travis-ci.org/sangria-graphql/sangria-msgpack.svg?branch=master)](https://travis-ci.org/sangria-graphql/sangria-msgpack) [![Coverage Status](http://coveralls.io/repos/sangria-graphql/sangria-msgpack/badge.svg?branch=master&service=github)](http://coveralls.io/github/sangria-graphql/sangria-msgpack?branch=master) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-msgpack_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-msgpack_2.11) [![License](http://img.shields.io/:license-Apache%202-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt) [![Join the chat at https://gitter.im/sangria-graphql/sangria](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/sangria-graphql/sangria?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://travis-ci.org/sangria-graphql-org/sangria-msgpack.svg?branch=master)](https://travis-ci.org/sangria-graphql-org/sangria-msgpack) [![Coverage Status](http://coveralls.io/repos/sangria-graphql-org/sangria-msgpack/badge.svg?branch=master&service=github)](http://coveralls.io/github/sangria-graphql-org/sangria-msgpack?branch=master) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-msgpack_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-msgpack_2.11) [![License](http://img.shields.io/:license-Apache%202-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt) [![Join the chat at https://gitter.im/sangria-graphql/sangria](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/sangria-graphql/sangria?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
for info, we can break them on different lines as markdown will still display them next to each other.
It makes clearer diff IMO.
(just for info)

Reply:
Agreed, and done.